### DISCUSSION TITLE
Pixi-Particles

### DISCUSSION THREAD

**Discussion by ghost** [pixijs/pixijs#9028] [Q&A] [answered]
Anyone know if Pixi-Particles works with v7? Is it still being developed/supported? Looking for a particles solution (and also a tween solution).

**Comment by SuperSodaSea** [comment, marked as answer]
See [v7-Migration-Guide](https://github.com/pixijs/pixijs/wiki/v7-Migration-Guide#%EF%B8%8F-plugin-supported):

> | Plugin | Compatible | Plugin Version Supported |
> |---|---|---|
> | [PixiJS Particle Emitter](https://github.com/pixijs/particle-emitter) | ✅ | v5.0.8+ |
